src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -48,16 +60,16 @@ public DissolutionCreateResponse submitDissolutionRequest(
             @Valid @RequestBody final DissolutionCreateRequest body,
             HttpServletRequest request) {
 
+        logger.debug("[POST] Submitting dissolution request for company number {}", companyNumber);
+
         if (StringUtils.isBlank(userId)) {
-            throw new UnauthorisedException();
+            throw new UnauthorisedException("ERIC headers are missing");
```

Comment:
Do you even need the messages? Doesn't the swagger docs account for this already? 404s and 401s are pretty self explanatory tbh. 400s are a bit more generic so could keep the message for them.

Reply:
Honestly, I was not thinking about Swagger at the time at all. I added them to enrich logging in our application - with these messages we actually know what has gone wrong in the logs, rather than generic 400 status.

Reply:
Won't we get that from URL + status code combination? i.e. GET /dissolution-request/1234 + 404 = no dissolution found for company number 1234

Reply:
We can get a bad request from a malformed request, or we can get a bad request from wrong application status.

Reply:
Yup - that's why I suggested keeping the error messages for bad requests but removing them for 404s and 401s :) See first comment in this thread ^

Reply:
Since you insist, I will remove them.